assets/dev/js/editor/components/ai/components/upgrade-chip.js

```diff
@@ -0,0 +1,136 @@
+import { useState } from 'react';
+import {
+	Chip as ChipBase,
+	Box,
+	Typography,
+	styled,
+	Popper,
+	Button,
+	List,
+	ListItem,
+	ListItemText,
+	ListItemIcon,
+} from '@elementor/ui';
+import { UpgradeIcon, CheckedCircleIcon } from '@elementor/icons';
+
+const popoverId = 'upgrade-popover';
```

Comment:
maybe make it more specific, since it gonna be the actual id in the HTML

```suggestion
const popoverId = 'e-ai-upgrade-popover';
```